### PR TITLE
User create booking

### PR DESCRIPTION
**Pull request by phucynwa**

### CODE REVIEW

**Comment by phucynwa** [comment]
https://dev.framgia.com/issues/184333

**Comment by phucynwa** [comment]
ready

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  before_action :logged_in_user
+
+  def new
+    @booking = Booking.new
+  end
+
+  def create
+    @booking = Booking.new booking_params
+    @booking.user = current_user
+    @booking.rooms << Room.find(params[:rooms])
```
Comment (huongnguyenmta) [inline app/controllers/bookings_controller.rb]: dùng transaction đi em ei

**Comment by huongnguyenmta** [comment]
commented

**Comment by phucynwa** [comment]
ready

**Pull request merged**